metrics/metrics.go

```diff
@@ -47,6 +48,7 @@ func InitOrError(m interface{}, factory Factory, globalTags map[string]string) e
 	counterPtrType := reflect.TypeOf((*Counter)(nil)).Elem()
```

Comment:
maybe we should fix #64 while at it

Reply:
I think InitOrError should be renamed to Init()

Reply:
done